src/lib/Sympa/Archive.pm

```diff
@@ -215,9 +215,16 @@ sub html_fetch {
     return undef unless $self->{arc_directory};
     return undef unless $options{file};
 
-    my $handle =
-        IO::File->new($self->{arc_directory} . '/' . $options{file}, '<');
-    return undef unless $handle;
+    my $html_file = $self->{arc_directory} . '/' . $options{file};
+    my $handle = IO::File->new($html_file, '<');
```

Comment:
I don't know why older men used `IO::File` instead of plain file handle. Though there seems no problem...

Reply:
Yes, it would make sense to always use the same pattern. But that's outside the scope of this pull request.